f8a_worker/graphutils.py

```diff
@@ -177,7 +177,8 @@ def aggregate_stack_data(stack, manifest_file, ecosystem):
             "total_licenses": len(stack_distinct_licenses),
             "distinct_licenses": list(stack_distinct_licenses),
             "popularity": stack_popularity_data,
-            "components": components
+            "components": components,
+            "manifest_file_path": manifest_file_path
```

Comment:
This is a schema change and can have widespread effect. So, we have to undergo discussions for such changes :-)

For now, please prefix the manifest_file_path with the manifest_name and provide that as the value for the "manifest_name" key.